explainer.md

```diff
@@ -345,6 +341,114 @@ And here's the result:
 
 https://user-images.githubusercontent.com/93594/184120371-678f58b3-d1f9-465b-978f-ee5eab73d120.mp4
 
+## Cross-document same-origin transitions
+
+This section outlines the navigation specific aspects of the ViewTransition API. The rendering model for generating snapshots and dispalying them using a tree of targetable pseudo-elements is the same for both SPA/MPA.
+
+### Declarative opt-in to transitions
+
+The first step is to add a new meta tag to the old and new Documents. This tag indicates that the author wants to enable transitions for same-origin navigations to/from this Document.
+
+```html
+<meta name="view-transition" content="same-origin">
```

Comment:
Is there a technical reason we couldn't have this signal in the CSS? It feels like it would benefit from being wrapped in media queries, particularly `prefers-reduced-motion`.

Reply:
The prefers-reduced-motion query is a good point. We can make this work in CSS.

Where should this CSS property be though? It's sort of a global knob so doesn't make sense for it to be a per-element property that participates in the style cascade.

Reply:
Discussed offline. We'll file an issue to bring this up with CSSWG. The media query for prefers-reduced-motion is an example for why it's better for it to be in CSS but we need to establish a pattern for global CSS settings.